src/pages/Welcome/Welcome.tsx

```diff
@@ -30,35 +25,22 @@ export const Welcome = () => {
             >
               <Button>{words.signin}</Button>
```

Comment:
Nesting button and link is not valid, we should style link as button or use button with onClick(() => {...some naviagtion...}) instead Link